src/GeoJson.js

```diff
@@ -161,6 +162,11 @@ ms.geoJson = function(fetch) {
     if (scale == "fixed") {
       initialScale = "scale(" + Math.pow(2, tile.zoom - (tile.scale = geoJson.map().zoom())) + ")";
     }
+    else if (scale == "relative") {
+      if(! zoomWhenLoaded) zoomWhenLoaded = tile.zoom;
+      tile.scale = geoJson.map().zoom();
+      initialScale = "scale(" + Math.pow(.5, zoomWhenLoaded - tile.zoom)  + ")";
```

Comment:
I think lines 167 and 168 are functionally identical to line 163. 
Also, if `tile.zoom == 0` the first time then  `!zoomWhenLoaded` will evaluate to true the next time around.